ush/eva/gen_eva_obs_yaml.py

```diff
@@ -7,7 +7,7 @@
 import yaml
 
 
-def gen_eva_obs_yaml(inputyaml, templateyaml, outputdir):
+def gen_eva_obs_yaml(inputyaml, templateyaml, outputdir, group, variable, bound):
```

Comment:
`group, variable, bound(s)` should be optional and set with a default value that would return what's currently in develop.